internal/pkg/aws/ecs/ecs.go

```diff
@@ -7,16 +7,18 @@ package ecs
 import (
 	"fmt"
 	"strings"
+	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/arn"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/dustin/go-humanize"
 )
 
 const (
-	shortTaskIDLength      = 7
-	shortImageDigestLength = 7
+	shortTaskIDLength      = 8
+	shortImageDigestLength = 8
```

Comment:
Just curious, is there a reason why you increased it?

Reply:
because previously these consts were defined in both two package (in this package these consts were actually unused and were both 7). Then I moved the code logic that use these consts to this package so I have to change the values for these.